doc/release-notes-15620.md

```diff
@@ -0,0 +1,11 @@
+Updated RPCs
+------------
+
+* The `sendrawtransaction` and `testmempoolaccept` previouly accepted an
```

Comment:
In commit "doc: Add release notes for 15620" (faadaae4795038d77936f36483f1410cd92b3238)

s/previouly accepted/RPC methods previously accepted/